ch6/test_util.py

```diff
@@ -0,0 +1,21 @@
+def fraction_equality(self, other):
+    return (
+        True
+        if self.numerator == other.numerator and self.denominator == other.denominator
+        else False
+    )
+
+
+def test61(examples):
```

Comment:
I'd like to push back on having a unique test runner for each individual problem. Maybe we need it and if we do that's fine, but the logic in a lot of these test methods looks very similar. My cursory assessment is that we could have a file in /materials with a small collection of test methods in it, which we could import as needed into each of the homework notebooks.